macaroons.go

```diff
@@ -140,11 +140,11 @@ var (
 		}},
 	}
 
-	// allPermissions is the list of all existing permissions that exist
+	// AllPermissions is the list of all existing permissions that exist
 	// for poold's RPC. The default macaroon that is created on startup
 	// contains all these permissions and is therefore equivalent to lnd's
 	// admin.macaroon but for pool.
-	allPermissions = []bakery.Op{{
+	AllPermissions = []bakery.Op{{
```

Comment:
Why/where is this needed? Should be its own commit with a short explanation in the commit message.